src/fstests/runner.py

```diff
@@ -67,60 +67,42 @@ def _run_single_job(self, tarball_node, device):
         finally:
             return True
 
-    def check_kvm_xfstest_env(self):
-        try:
-            result = subprocess.run(
-                "kvm-xfstests --help", check=True, shell=True)
-            if result.returncode == 0:
-                print(f"KVM-xftests found.")
-                return True
-            else:
-                return False
-        except Exception as e:
-            print(f"Raised exception: {e}")
+    def run(self, args):
+        if args.node_id:
+            tarball_node = self._db.get_node(args.node_id)
+        else:
+            tarball_node = None
+        if tarball_node is None:
+            print("Node not found")
             return False
-
-    def run(self):
-        sub_id = self._db.subscribe_node_channel({
-            'op': 'created',
-            'name': 'tarball',
-        })
-        print('Listening for tarballs')
-        print('Press Ctrl-C to stop.')
-        try:
-            while True:
-                tarball_node = self._db.receive_node(sub_id)
-                print(f"Node tarball with id: {tarball_node['_id']}\
-                   from revision: {tarball_node['revision']['commit'][:12]}")
-                device = self._device_configs['shell']
-                self._run_single_job(tarball_node, device)
```

Comment:
Why are you removing this?  It's what the service should be doing, waiting for events about new tarball nodes.

The `--node-id` argument is optional and it's just a way of manually testing a particular tarball during development.